mail_tracking/models/mail_tracking_email.py

```diff
@@ -115,6 +116,14 @@ class MailTrackingEmail(models.Model):
         groups="base.group_system",
     )
 
+    @api.depends("mail_message_id")
+    def _compute_message_id(self):
+        """This helper field will allow us to map the message_id from either the linked
+        mail.message or a mass.mailing mail.trace"""
```

Comment:
Proper PEP257 format.

```suggestion
        mail.message or a mass.mailing mail.trace.
        """
```